types/nationalAvalancheCenter/schemas.ts

```diff
@@ -341,7 +343,22 @@ export const externalMediaSchema = imageMediaSchema.extend({
 });
 export type ExternalMediaItem = z.infer<typeof externalMediaSchema>;
 
-export const mediaItemSchema = z.discriminatedUnion('type', [emptyMediaSchema, nullMediaSchema, imageMediaSchema, videoMediaSchema, externalMediaSchema, photoMediaSchema]);
+export const pdfMediaSchema = z.object({
+  type: z.literal(MediaType.PDF),
+  url: z.object({
+    original: z.string().url(),
+  }),
+});
+
+const unknownMediaSchema = z.object({
+  type: z.literal(MediaType.Unknown),
+});
+
+export const mediaItemSchema = z
+  .discriminatedUnion('type', [emptyMediaSchema, nullMediaSchema, imageMediaSchema, videoMediaSchema, externalMediaSchema, photoMediaSchema, pdfMediaSchema, unknownMediaSchema])
+  .catch({
```

Comment:
Nice. I was looking at `.catchall()` at the global scope to see if we could do something like this for all of our enums but that looks like it only applies to unknown keys, not unknown values. Wonder if you think I should add one of these to the rest of our enums? And maybe in debug mode or cron mode we yell instead of falling back so we know when we need to update?